packages/terra-form-select/src/terra-dev-site/doc/form-select/Select.01.doc.mdx

```diff
@@ -47,7 +47,10 @@ This component requires the following peer dependencies be installed in your app
 
 The Form-Select component must be composed inside the [Base][1] component with a locale in order for it to load the correct translation strings.
 
+As a result of accessibility issues, the default implementation of select should no be used. It will be fully removed in a future major version bump. To achieve the same functionality the [Native Select][2] should be consumed.
+
 [1]: https://engineering.cerner.com/terra-core/components/terra-base/base/base
+[2]: https://engineering.cerner.com/terra-core/components/terra-form-select/form-select/native-select
```

Comment:
Doing this should allow this link to work for both terra-ui and terra-core
```suggestion
[2]: /components/terra-form-select/form-select/native-select
```

Reply:
Updated.